src/SyncTrayzor/Pages/AboutView.xaml

```diff
@@ -6,6 +6,7 @@
         Height="400" Width="250"
         WindowStyle="None" SizeToContent="Height"
         ShowInTaskbar="False"
+        ResizeMode="CanMinimize"
```

Comment:
They have the same result AFAIK, but having it on CanMinimize makes it easier to switch to a bordered window in the future if wanted.